integration/inspektor-gadget/integration_test.go

```diff
@@ -488,7 +506,9 @@ func TestDns(t *testing.T) {
 	commands := []*Command{
 		CreateTestNamespaceCommand(ns),
 		dnsCmd,
-		BusyboxPodRepeatCommand(ns, "nslookup inspektor-gadget.io 8.8.4.4"),
+		BusyboxPodRepeatCommand(ns,
+			"nslookup -type=a inspektor-gadget.io. 8.8.4.4 ;"+
+				"nslookup -type=aaaa inspektor-gadget.io. 8.8.4.4"),
```

Comment:
Why did you need to split the request, doesn't `nslookup` perform both requests by default?

Reply:
To guarantee the ordering of the 4 UDP packets: the busybox implementation of nslookup sends both requests without waiting for the responses. So depending how fast is the network, the first response could arrive between the two requests.

The nslookup implementation from bind-utils does not do that.